frontend/src/bundles/candidate/pages/candidate.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button, Grid } from '~/bundles/common/components/components.js';
+import { useCallback, useState } from '~/bundles/common/hooks/hooks.js';
+
+import { CandidateModal } from '../components/components.js';
+import styles from './styles.module.scss';
+
+// This page is for demonstrative purpose, fill with candidate page logic later
+const Candidate: React.FC = () => {
+    const [isModal1Open, setIsModal1Open] = useState(false);
+    const [isModal2Open, setIsModal2Open] = useState(false);
+
+    const closeModal1 = useCallback(() => {
+        setIsModal1Open(false);
+    }, []);
+    const openModal1 = useCallback(() => {
+        setIsModal1Open(true);
+    }, []);
+    const closeModal2 = useCallback(() => {
+        setIsModal2Open(false);
+    }, []);
+    const openModal2 = useCallback(() => {
+        setIsModal2Open(true);
+    }, []);
```

Comment:
Code duplication. In this case good practice to create hook for example useVisibility to manage modal isOpen state.

Reply:
This code is only for demonstration/testing of this component. Do I need to change this part if we will remove it before merging?